lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -716,17 +716,61 @@ struct RvalueExprVisitor {
                   const slang::ast::CallExpression::SystemCallInfo &info) {
     const auto &subroutine = *info.subroutine;
     auto args = expr.arguments();
+    auto value = context.convertRvalueExpression(*args[0]);
+    if (!value)
+      return {};
```

Comment:
We need to check whether `args.size() == 1` first. Maybe you could wrap the `convertRvalueExpression` and your entire StringSwitch in an `if (args.size() == 1) { ... }`? In the future, when we support different arities of system tasks/calls, we can expand this to call into some special handling functions like `convertSystemCall1`, `convertSystemCall2`, `convertSystemCallVariadic`, and similar. Not needed now though, since all we support are unary/arity-1 system calls that have a single argument.